apps/E2E/src/common/BasePage.ts

```diff
@@ -72,6 +72,22 @@ export class BasePage {
   /******************************************************************/
   /**************** UI Element Interaction Methods ******************/
   /******************************************************************/
+
+  /**
+   * Checks to see if an element attribute is strictly equal to an expected value the user passes in.
+   * The advantage to this over testing using .isEqual in a spec is that this throws a detailed error if
+   * the expected and actual values don't match. This should be called for attribute tests in specs. */
+  async compareAttribute(element: Promise<WebdriverIO.Element>, attribute: Attribute, expectedValue: any): Promise<boolean> {
+    const actualValue = await (await element).getAttribute(attribute);
+    if (expectedValue !== actualValue) {
+      throw new Error(
+        `On ${this._pageName}, a test component should have attribute, ${attributeToEnumName[attribute]} (${attribute}), equal to ${expectedValue}.
```

Comment:
Nit: One more improvement - I think the `${attributeToEnumName[attribute]} (${attribute})` could be a little confusing to a user. This expands to (for example), `AccessibilityLabel (Name)`. For someone who's not familiar, I would probably be confused at the multiple property names.

Let's instead clarify `${attributeToEnumName[attribute]} (which maps to the Windows '${attribute}' property on the Element)`

Reply:
Also, we should specify which element is in error.

Is it possible to get the testID/automationID from a WebDriverIO.Element? If so, we can also put that in this error message

Reply:
Doing a `getAttribute('AutomationId')`  should do the trick.